Converted samples to use dialog.open and dialog.submit [Don't Merge] (#762)
OfficeDev/Microsoft-Teams-Samples: samples/bot-task-module/csharp/README.md

---
page_type: sample

> description: This sample app demonstrate how to use Dialog using Bot Framework v4

Reviewer: ```suggestion
description: This sample app demonstrate how to use Dialogs (referred as task modules in TeamsJS v1.x) using Bot Framework v4
```

Reviewer 2: Updated

urlFragment: officedev-microsoft-teams-samples-bot-task-module-csharp

---

> # Teams Dialog

Reviewer: ```suggestion
# Teams Dialogs (referred as task modules in TeamsJS v1.x)
```

Reviewer 2: Updated

> Bot Framework Teams Dialog sample.

Reviewer: ```suggestion
Bot Framework Teams Dialogs (referred as task modules in TeamsJS v1.x) sample.
```

Reviewer 2: Updated

This bot has been created using [Bot Framework](https://dev.botframework.com). It shows how to fetch a Dialog from Hero Card or Adaptive Card buttons and receive input from the Dialog in the bot.

## Included Features
* Bots
* Tabs

> * Dialogs

Reviewer: ```suggestion
* Dialogs (referred as task modules in TeamsJS v1.x)
```

Reviewer 2: Updated

## Interaction with bot
![bot-task-module ](Images/Bot_Tab_TaskModule.gif)

## Try it yourself - experience the App in your Microsoft Teams client
Please find below demo manifest which is deployed on Microsoft Azure and you can try it yourself by uploading the app manifest (.zip file link below) to your teams and/or as a personal app. (Sideloading must be enabled for your tenant, [see steps here](https://docs.microsoft.com/microsoftteams/platform/concepts/build-and-test/prepare-your-o365-tenant#enable-custom-teams-apps-and-turn-on-custom-app-uploading)).

> **Teams Dialog:** [Manifest](/samples/bot-task-module/csharp/demo-manifest/bot-task-module.zip)

Reviewer: ```suggestion
**Teams Dialogs (referred as task modules in TeamsJS v1.x):** [Manifest](/samples/bot-task-module/csharp/demo-manifest/bot-task-module.zip)
```

Reviewer 2: Updated
